runatlantis.io/docs/server-configuration.md

```diff
@@ -886,6 +886,14 @@ and set `--autoplan-modules` to `false`.
   ```
   File containing x509 private key matching `--ssl-cert-file`.
 
+### `--restrict-file-list`
+  ```bash
+  atlantis server --restrict-file-list
+  # or (recommended)
+  ATLANTIS_RESTRICT_FILE_LIST=true
+  ```
+  `--restrict-file-list` will block plan requests from projects outside the files modified in the pull request.
```

Comment:
Will this command restrict the ability to use the `--enable-regexp-cmd` flag? If so, should we call it out here in the docs?

Reply:
I believe that as long the files are in the PR change list this should be fine, I'll test this anyway.

Reply:
Oh, I didn't knew it was used for this, I'll test to just to get the exact results to update the documentation anyway.

Reply:
Looks like the `--enable-regexp-cmd` bypasses the checks. I'm unsure of how it does this, I not familiarized with this code, which should be the expected behavior? Bypass the check or not?

Tested here: https://github.com/Fabianoshz/atlantis-test/pull/6

Reply:
That's ok. Let's just document this for now that the --enable-regexp-cmd will allow bypassing the --restrict-file-list flag. Id mention this under both flags in the server config.

Reply:
Ok, fine by me, I'll this to the documentation.